docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '2'
+
+volumes:
+    data:
+        driver: local
+    cache:
+        driver: local
+    logs:
+        driver: local
+
+services:
+    sylius:
+        build:
+            context: .
+            dockerfile: ./etc/docker/php/Dockerfile
+        entrypoint: /var/www/etc/docker/php/entrypoint.sh
+        volumes:
+            - .:/var/www
+            - data:/var/www/sylius
+            - cache:/var/www/app/cache
+            - logs:/var/www/app/logs
+        depends_on:
+            - database
```

Comment:
Yes, `version: "2"` syntax creates a network for each service in our application and they can communicate with each other using the service name as a host. No linking required.

`depends_on` just tells Docker that sylius needs that container/hostname to exist to start